src/form/bf_fbs.jl

```diff
@@ -79,6 +79,12 @@ function variable_mc_bus_voltage(pm::FBSUBFPowerModel; nw=nw_id_default, bounded
 end
 
 
+"Capacitor switching variables"
```

Comment:
should correct to follow new conventions for docstring format that include the function signature